operator/controllers/csiscaleoperator_controller.go

```diff
@@ -219,6 +221,74 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 
 	logger.Info("Creation of the resources which never change is successful")
 
+	// Rollout restart of node plugin pods, if modified driver
+	// manifest file is applied.
+	// The CSIConfigMap has the data which is last applied, so
+	// compare the clusters data of current driver instance with
+	// CSIConfigMap data and rollout restart if there is a difference.
+	configMap := &corev1.ConfigMap{}
+	cerr := r.Client.Get(context.TODO(), types.NamespacedName{
+		Name:      config.CSIConfigMap,
+		Namespace: req.Namespace,
+	}, configMap)
+	if cerr != nil {
+		if !errors.IsNotFound(cerr) {
+			message := "Failed to get ConfigMap resource " + config.CSIConfigMap
+			logger.Error(cerr, message)
+			// TODO: Add event.
+			meta.SetStatusCondition(&crStatus.Conditions, metav1.Condition{
+				Type:    string(config.StatusConditionSuccess),
+				Status:  metav1.ConditionFalse,
+				Reason:  string(csiv1.ResourceReadError),
+				Message: message,
+			})
+			logger.Error(err, message)
```

Comment:
`err` name is not defined, can replace all `cerr` with `err`? `cerr` is not very descriptive, you may change it to just `err` .

Reply:
Removed this additional log, we already have this error logged. Golang recommends short variable names, so I have used `derr` for error getting daemoset and `cerr` for error getting configmap. Please do not block the merge for variable names!  And we cannot use the same error name `err` in all the places, e.g. we cannot use the same variable `err` instead of `derr` here:
```
daemonSet, derr := r.getNodeDaemonSet(instance)
```
There is another way of declaring `daemonSet` separately and using the same var `err` in the another statement, but I prefer to write less code which does the same job wherever possible.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#variable-names